.gitignore

```diff
@@ -1,4 +1,5 @@
-.*
+.DS_Store
+.devcontainer
```

Comment:
This is already checked in.
If you want to get rid of it, we should remove the files it as well.

```suggestion
```

Reply:
sure, let's keep `.devcontainer`. It was excluded by `.*` though